### PR TITLE
feat: add vmtests

### PR DESCRIPTION
**Pull request by greged93**
Adds the `VMTests` to the current runner, with all skipped tests linked to a PR.
Time spent on this PR: 3 days

Resolves: #28 

# Pull Request type

Please check the type of change your PR introduces:

-   [ ] Bugfix
-   [x] Feature
-   [ ] Code style update (formatting, renaming)
-   [ ] Refactoring (no functional changes, no API changes)
-   [ ] Build-related changes
-   [ ] Documentation content changes
-   [ ] Testing

# What is the new behavior?
Runner can run `VMTests`.

# Does this introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

File: crates/ef-testing/src/models/case.rs [hunk 1]
```diff
@@ -66,7 +66,28 @@ impl BlockchainTestCase {
...
+                | "jumpToPush.json" // ef-tests #61
+                | "signextend.json" // kakarot #677
+                | "mload.json" // ef-tests #31
+                | "gas.json" // ef-tests #36
+                | "loopExp.json" // ef-tests #39
+                | "loopMul.json" // ef-tests #39
+                | "performanceTester.json" // ef-tests #39
+                | "suicide.json" // ef-tests #57
+                | "blockInfo.json" // ef-tests #67
+                | "envInfo.json" // ef-tests #63
```
Comment (ftupas) [inline crates/ef-testing/src/models/case.rs:90]: what are these numbers exactly?

File: crates/ef-testing/tests/tests.rs [hunk 2]
```diff
@@ -2,44 +2,16 @@
...
-                local_sha, remote_sha
-            )));
-        }
-
-        Ok(remote_sha)
-    }
-
-    #[ctor]
-    fn setup() {
+fn set_up() {
```
Comment (ftupas) [inline crates/ef-testing/tests/tests.rs:11]:
```suggestion
fn setup() {
```

File: crates/ef-testing/src/models/case.rs (hunk 1 above)
Comment (greged93) [reply crates/ef-testing/src/models/case.rs:90]: issue number related to why we skip the tests for now

Comment (ftupas) [reply crates/ef-testing/src/models/case.rs:90]: Gotcha thanks!

**Review by ftupas — APPROVED** [review]
Looks good to me

**Pull request merged**